client/src/components/downloadPacket.module.scss

```diff
@@ -0,0 +1,49 @@
+$primary-color: #112f4e;
+
+.downloadBoxContainer {
+    flex: 0 0 350px;
+    // flex: 1;
+    // border: 1px solid red;
+
+    color: whitesmoke;
+    margin: auto;
+
+    .downloadBox {
+      background-color: $primary-color;
+      min-width: 400px;
+      border-radius: 6px 6px;
```

Comment:
This goes for all pixel values in this file actually - could these be ems or percentages to ensure [responsiveness](https://dev.to/j836/font-size-em-rem-pixel-or-percent-3f3p)? I realize this holds for all of our existing css files also but we should have been more diligent!

Reply:
May make sense to handle this as part of the mobile audit actually - better to merge as-is and make a followup ticket to nix all `px` across the site, we've got quite a few